main/menus.js

```diff
@@ -20,6 +20,32 @@ const checkForUpdatesItem = {
   }
 };
 
+const issueBody = `
+<!--
+Thank you for helping us test the Kap Beta, your feedback helps us make Kap better for everyone!
```

Comment:
I think we should still have the feedback button even after the beta, so let's just use `Kap` and not `Kap Beta`. Alternatively, add a conditional.